crates/alpen-ee/exec-chain/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+#![expect(unused_crate_dependencies, reason = "wip")]
```

Comment:
Can we remove this before we merge? If no, please add a `// TODO(@sapinb): ...` next to it?